app/views/coronavirus_form/cookies.html.erb

```diff
@@ -38,36 +38,31 @@
     <% end %>
   </div>
 
-  <div class="cookie-settings__form-wrapper">
-    <p>
-      We use <%= t('cookies.settings_page.cookies').count %> types of cookie.
-      You can choose which cookies you're happy for us to use.
-    </p>
+  <% t('cookies.settings_page.tables').map do |table| %>
+    <div class="govuk-body govuk-!-padding-top-6">
```

Comment:
We don't need `govuk-body` here as it's on the paragraphed element.

```suggestion
    <div class="govuk-!-padding-top-6">
```

It's worth seeing if we can remove the override class, since we try and avoid using them if possible.